interface/orders/order_manifest.php

```diff
@@ -178,162 +178,159 @@ function generate_order_summary($orderid)
+            <td width='45%'><?php echo myCellText($patient_id); ?></td>
+            <td width='5%' nowrap><?php echo xlt('Order ID'); ?></td>
+            <td width='45%'><?php echo myCellText($orow['procedure_order_id']); ?></td>
+        </tr>
+        <tr>
+            <td nowrap><?php echo xlt('Encounter Date'); ?></td>
+            <td><?php echo myCellText(oeFormatShortDate($encdate)); ?></td>
+            <td nowrap><?php echo xlt('Order Date'); ?></td>
+            <td><?php echo myCellText(oeFormatShortDate($orow['date_ordered'])); ?></td>
+        </tr>
+        <tr>
+            <td nowrap><?php echo xlt('Birth Date'); ?></td>
+            <td><?php echo myCellText(oeFormatShortDate($orow['DOB'])); ?></td>
+            <td nowrap><?php echo xlt('Lab'); ?></td>
+            <td><?php echo myCellText($orow['labname']); ?></td>
+        </tr>
+        <tr>
+            <td nowrap><?php echo xlt('Sex'); ?></td>
+            <td><?php echo myCellText(getListItem('sex', $orow['sex'])); ?></td>
+            <td nowrap><?php echo xlt('Specimen Type'); ?></td>
+            <td><?php echo myCellText($orow['specimen_type']); ?></td>
+        </tr>
+        <tr>
+            <td nowrap><?php echo xlt('Address'); ?></td>
+            <td><?php echo myCellText($orow['street'] . ', ' . $orow['city'] . ', ' . $orow['state'] . ' ' . $orow['postal_code']); ?></td>
+            <td nowrap><?php echo xlt('Collection Date'); ?></td>
+            <td><?php echo myCellText(oeFormatShortDate($orow['date_collected'])); ?></td>
+        </tr>
+        <tr>
+            <td nowrap><?php echo xlt('Ins Name'); ?></td>
+            <td><?php echo myCellText($ins_name); ?></td>
+            <td nowrap><?php echo xlt('Clinical History'); ?></td>
+            <td><?php echo myCellText($orow['clinical_hx']); ?></td>
+        </tr>
+        <tr>
+            <td nowrap><?php echo xlt('Ins Address'); ?></td>
+            <td><?php echo myCellText("$ins_addr, $ins_city, $ins_state $ins_zip"); ?></td>
+            <td nowrap><?php echo xlt('Order Status'); ?></td>
+            <td><?php echo myCellText(getListItem('ord_status', $orow['order_status'])); ?></td>
+        </tr>
+        <tr>
+            <td nowrap><?php echo xlt('Ins Policy'); ?></td>
+            <td><?php echo myCellText($ins_policy); ?></td>
+            <td nowrap><?php echo xlt('Referrer'); ?></td>
+            <td><?php echo myCellText($orow['ref_lname'] . ', ' . $orow['ref_fname'] . ' ' . $orow['ref_mname']); ?></td>
+        </tr>
+        <tr>
+            <td nowrap><?php echo xlt('Ins Group'); ?></td>
+            <td><?php echo myCellText($ins_group); ?></td>
+            <td nowrap><?php echo xlt('Referrer Address'); ?></td>
+            <td><?php echo myCellText($orow['ref_street'] . ', ' . $orow['ref_city'] . ', ' . $orow['ref_state'] . ' ' . $orow['ref_zip']); ?></td>
+        </tr>
+    </table>
+
+    <br />
+
+    <table class='table'>
+        <tr class='head'>
```

Comment:
Add a ` <thead>` element before and after the table header. Also change the `<td>` elements to `<th>`.